apps/discovery_api/runtime.exs

```diff
@@ -231,3 +233,7 @@ config :telemetry_event,
 if System.get_env("RAPTOR_URL") do
   config :discovery_api, raptor_url: System.get_env("RAPTOR_URL")
 end
+
+if System.get_env("RAPTOR_LIST_URL") do
+  config :discovery_api, raptor_url: System.get_env("RAPTOR_LIST_URL")
+end
```

Comment:
We'll need to add `raptor_list_url` to the discovery api chart as part of this work too right? So that it can be set here like RAPTOR_URL is?

At one point we mentioned potentially refactoring to only need one raptor config URL. Needing to update the chart as a consequence of not doing that makes me think the refactor might be beneficial.

Idea for the refactor:
Could we just give the RaptorService the `raptor_url()` in all cases, and it could append the `/listAccessGroups` or `/authorize` route onto the URL as needed on a method by method basis? If we went with that refactor, we'd ironically have to change the chart anyway because the default raptor endpoint includes `/authorize` and we'd need to remove that to make my refactor suggestion work.

Not attached to any ideas but figured I'd throw that out there.